sql/data_model.sql

```diff
@@ -0,0 +1,43 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
```

Comment:
In Postgres out doesn't matter if you use text or varchar. The only thing
varchar does is so an extra check at insert time to verify length.